app/components/wallet/hwConnect/common/ConnectDialog.scss

```diff
@@ -16,6 +16,14 @@
       max-height: 100%;
     }
   }
+
+  & + .Dialog_title {
+    margin-bottom: 12px !important;
+  }
+
+  & + .Dialog_actions {
+    margin-top: 24px !important;
+  }
```

Comment:
@leivaburto 

```
  & + .Dialog_title {
    margin-bottom: 12px !important;
  }

  & + .Dialog_actions {
    margin-top: 24px !important;
  }
```

is this block still needed? 
the same logic is at the bottom of the file